eng/pipelines/cadl-ci.yml

```diff
@@ -76,6 +76,11 @@ stages:
             command: custom
             customCommand: pack
             workingDir: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
+        - task: Npm@1
```

Comment:
Install cadl libraries under autoreset.csharp root directory which is needed when generate the test projects because the work directory is changed to root directory.

Reply:
If so, I believe we could remove the step of "installing dependencies under emitter.csharp"?